server/controllers/submit-controller.js

```diff
@@ -23,7 +23,7 @@ module.exports = ({ strapi }) => ({
     for (const provider of strapi.config.get('plugin.ezforms.notificationProviders')) {
       if (provider.enabled) {
         try {
-          await strapi.plugin('ezforms').service(provider.name).send(provider.config, ctx.request.body.formData)
+          await strapi.plugin('ezforms').service(provider.name).send(provider.config, strapi.config.get('plugin.ezforms.enableFormName') === true ? ctx.request.body.formName : "form", ctx.request.body.formData)
```

Comment:
I think it would be better to just store this in a variable in this file then we don't need to have duplicated ternary operators. Also we don't need the === true